packages/shared-ux/chrome/navigation/src/ui/hooks/use_init_navnode.ts

```diff
@@ -59,7 +59,7 @@ function getNodeStatus(
     if (!hasUserAccessToCloudLink()) return 'remove';
   }
 
-  if (deepLink && deepLink.hidden) return 'remove';
+  if (deepLink && deepLink.hidden) return 'hidden';
```

Comment:
@sebelga, the newly added 

```
{
               link: 'observability-overview:cases_configure',
             },
             {
               link: 'observability-overview:cases_create',
             },
```

are used for the breadcrumbs, but their deep links are "hidden". 

The fix I did is to keep such links in the navigation tree for the breadcrumbs, but to filter them out when rendering the side nav.

Reply:
Ok 👍